exercises/.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "singleQuote": true,
+  "semi": false,
```

Comment:
I would go with semicolons. :slightly_smiling_face:

Reply:
please noooooo! :)

Reply:
> please noooooo! :)

It's a strong opinion :smile: 

@branoholy Would you mind if the semicolons were removed?

Reply:
I personally prefer semicolons so I would vote for them but I'm OK with both options. The most important thing is that it's set in the prettier config. :+1: